frontend/appflowy_flutter/lib/plugins/ai_chat/chat_page.dart

```diff
@@ -273,10 +278,26 @@ class _ChatContentPage extends StatelessWidget {
           chatId: view.id,
           refSourceJsonString: refSourceJsonString,
           isLastMessage: isLastMessage,
-          onSelectedMetadata: (metadata) {
-            context
-                .read<ChatSidePanelBloc>()
-                .add(ChatSidePanelEvent.selectedMetadata(metadata));
+          onSelectedMetadata: (metadata) async {
+            if (isURL(metadata.name)) {
```

Comment:
What do you think about putting the open link logic in the bloc?

Reply:
This code will be changed in a following PR because the chat side panel bloc will be gone.